virtualbox/machine.go

```diff
@@ -690,13 +690,13 @@ func (m *Machine) Modify() error {
 
 // AddNATPF adds a NAT port forarding rule to the n-th NIC with the given name.
 func (m *Machine) AddNATPF(n int, name string, rule driver.PFRule) error {
-	return vbm("controlvm", m.Name, fmt.Sprintf("natpf%d", n),
+	return vbm("modifyvm", m.Name, fmt.Sprintf("--natpf%d", n),
```

Comment:
Ideally, the diff in this PR looks as much like the diff in https://github.com/docker/machine/pull/1496 as we can and actually have VBox 5 "work".